### PR TITLE
Add measured pathDB

### PR DESCRIPTION
**Pull request by ruhan1**
Add a measurable pathDB impl.

### CODE REVIEW

File: src/main/java/org/commonjava/storage/pathmapped/metrics/MeasuredPathDB.java [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+
+    private static final String TIMER = "timer";
+
+    private static final String METER = "meter";
+
+    private void measure( Runnable runnable, String metricName )
+    {
+        if ( metricRegistry != null )
+        {
+            metricRegistry.meter( name( metricPrefix, metricName, METER ) ).mark();
```
Comment (ligangty) [inline src/main/java/org/commonjava/storage/pathmapped/metrics/MeasuredPathDB.java]: A timer should already involve a meter, so I'm thinking if this meter is still needed here.

File: src/main/java/org/commonjava/storage/pathmapped/metrics/MeasuredPathDB.java [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+        }
+    }
+
+    private <T> T measure( Callable<T> callable, String metricName )
+    {
+        try
+        {
+            if ( metricRegistry != null )
+            {
+                metricRegistry.meter( name( metricPrefix, metricName, METER ) ).mark();
```
Comment (ligangty) [inline src/main/java/org/commonjava/storage/pathmapped/metrics/MeasuredPathDB.java]: See above.

Comment (ruhan1) [reply src/main/java/org/commonjava/storage/pathmapped/metrics/MeasuredPathDB.java]: +1. I removed the meter.

**Review by ligangty — APPROVED** [review]
LGTM

**Pull request merged**